configs/AWS-Hpc7a-EPYC4-EFA/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: AWS-Hpc7a-EPYC4-EFA
+  manufacturer:
+    vendor: AWS
+    name: ParallelCluster3.7.2-Hpc7a
+  processor:
+    vendor: AMD
+    name: EPYC-Zen4
+    ISA: x86_64
+  accelerator:
+    vendor: ""
+    name: ""
+    GPU_ISA: ""
+  interconnect:
+    vendor: AWS
+    name: EFA
+  OS:
+    name: Amazon-Linux-2
+  system-tested:
+    owner: AWS
+    name: ""
+    installation-year: ""
+    description: [aws/hpc7a](https://aws.amazon.com/ec2/instance-types/hpc7a)
```

Comment:
Ideally we could get a github gist here with the yaml description for the pcluster setup -- that way it would be reproducible for the cloud workflows.

Reply:
yes!  A cluster setup yaml is definitely something we want to keep.  Do you think we would ask people to add the yaml file in the system definition directory?  Or do you mean something else by GitHub gist?  I do think the system_definition.yaml should point at it - suggestions for how to name the field?